### ISSUE TITLE
Swaps - Uncaught TypeError: e.slice is not a function

### ISSUE THREAD

**Issue by PeterYinusa** [MetaMask/metamask-extension#12121]
**Describe the bug**
User's cannot update the gas fee in the legacy customize gas modal under the following conditions:

- Non EIP supported network
- Non USD Currency conversion

**Steps to reproduce**

1. Login to the extension
2. Switch to `Binance Smart Chain` network
3. Change Currency Conversion to `GBP - Pound Sterling`
4. Click Swap on the Home page, complete the build a quote form and click `Review Quote`
5. Click the `Edit` link to the right of Max fee
6. Switch to the `Advanced` tab of the Customize Gas modal
7. Change the `Gas Price` and click `Save`

**Expected behavior**
- User can update the gas fee
- No uncaught exceptions

**Error Message**
- Uncaught TypeError: e.slice is not a function

```
TypeError: e.slice is not a function
  at onSubmit(metamask/ui-3.js:1:968241)
  at onClick(metamask/ui-3.js:1:832144)
  at Object.h(metamask/ui-0.js:1:29516)
  at k(metamask/ui-0.js:1:29659)
  at ? (metamask/ui-0.js:1:29805)
  at x(metamask/ui-0.js:1:29891)
  at C(metamask/ui-0.js:1:30317)
  at P(metamask/ui-0.js:1:30152)
  at R(metamask/ui-0.js:1:30500)
  at En(metamask/ui-0.js:1:57481)
  at le(metamask/ui-0.js:1:142969)
  at Rn(metamask/ui-0.js:1:58096)
  at Mn(metamask/ui-0.js:1:59053)
  at Dn(metamask/ui-0.js:1:58341)
  at n.unstable_runWithPriority(metamask/ui-0.js:1:268259)
  at Ko(metamask/ui-0.js:1:74106)
  at se(metamask/ui-0.js:1:142746)
  at On(metamask/ui-0.js:1:57764)
  at HTMLDocument.a(metamask/sentry-install.js:1:10465)
```

**Comment by danjm** [comment]
@PeterYinusa Does this happen on rinkeby if you are not using a trezor?

**Comment by PeterYinusa** [comment]
> @PeterYinusa Does this happen on rinkeby if you are not using a trezor?

If you're not using a Trezor you will see the new Edit Priority dialog.

I imagine this can happen on Binance without a Trezor as the legacy Customize Gas modal is shown.

**Comment by AlexJupiter** [comment]
@PeterYinusa 
> I imagine this can happen on Binance without a Trezor as the legacy Customize Gas modal is shown.

Does this mean then this isn't a Trezor specific issue?

**Comment by PeterYinusa** [comment]
Fixed in 10.16.0
